doc/themes/the-things-stack/layouts/partials/header.html

```diff
@@ -1 +1,2 @@
 {{- partial "nav.html" . -}}
+{{- partial "intercom.html" . -}}
```

Comment:
Unless intercom recommends otherwise, this should go into the footer partial, so it doesn't slow down the DOM rendering.